app/assets/stylesheets/modules/show_playdates.scss

```diff
@@ -0,0 +1,27 @@
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+#map {
+  width: 450px;
```

Comment:
Properties should be ordered border-style, border-width, box-shadow, height, width